src/utils/formUtil.ts

```diff
@@ -193,6 +193,12 @@ export function getModelType(component: Component): keyof typeof MODEL_TYPES_OF_
     return component.modelType;
   }
 
+  switch (component.type) {
```

Comment:
I think I'd rather just treat textarea components like selects and make their data model just 'any' to begin with. Also, see my comment in the [upstream formio.js PR](https://github.com/formio/formio.js/pull/5829/files)

Reply:
I have changed the textarea data model to 'any' and added normalization logic for it in the renderer.
I also added a fix for the issue where the validation error appears for multiple textarea with JSON type when the first value is array.